src/components/BoardUsers/BoardUsers.tsx

```diff
@@ -0,0 +1,32 @@
+import BoardUser from './BoardUser';
+import RestUserCounter from './RestUsersCounter';
+import './BoardUsers.scss';
+import Parse from 'parse';
+import { useSelector } from 'react-redux';
+import { ApplicationState } from 'types/store';
+
+export interface BoardUsersProps {
+  numOfUsersToShow: number;
+}
+
+const BoardUsers = ({ numOfUsersToShow }: BoardUsersProps) => {
+  const users = useSelector((state: ApplicationState) => state.users.all);
+  const currentUser = Parse.User.current();
+
+  const me = users.find(user => user.id === currentUser!.id);
+  const connectedUsers = users.filter(user => user.online);
+  const them = connectedUsers.filter(user => user.id !== currentUser!.id);
+  const usersToShow = them.splice(0, (them.length > numOfUsersToShow ? numOfUsersToShow - 1 : numOfUsersToShow));
+
+  return (
+    <div className="users">
+      <ul className="user-list">
+        {them.length > 0 && <RestUserCounter count={them.length} />}
```

Comment:
I think the `RestUserCounter` component is unnecessary. The overhead of a new component seems to me to be much more that those 4 lines of code put here

Reply:
done `RestUserCounter` removed